src/main/java/com/traveltime/sdk/dto/requests/protodistance/Transportation.java

```diff
@@ -0,0 +1,15 @@
+package com.traveltime.sdk.dto.requests.protodistance;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+
+@Getter
+@AllArgsConstructor
+public enum Transportation {
+    DRIVING_FERRY("driving+ferry", 3),
+    WALKING("walking", 4),
+    DRIVING("driving", 1);
+
```

Comment:
Right now we only have 
```
driving+ferry
walking+ferry
```
You can't separate just driving or walking, right @bartosz-witkowski?

Reply:
We can and we even have maps for it - they're just not deployed.

Reply:
should I leave them?

Reply:
We don't have it for distance, please leave only `driving+ferry` and `walking+ferry`. If we add them in the future we will update SDK.

Reply:
done